CHANGELOG.md

```diff
@@ -14,6 +14,9 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - Add `RecordFactory` in `go.opentelemetry.io/otel/sdk/log/logtest` to facilitate testing the exporter and processor implementations. (#5258)
 - Add example for `go.opentelemetry.io/otel/exporters/stdout/stdoutlog`. (#5242)
 - The count of dropped records from the `BatchProcessor` in `go.opentelemetry.io/otel/sdk/log` is logged. (#5276)
+- Add the synchronous gauge instrument to `go.opentelemetry.io/otel/metric`. (#5304)
```

Comment:
(non-blocking)

What do you think of calling out all the added exported API? I mean do you think that it is worth mentioning that this PR:
- adds `Int64Gauge` and `Float64Gauge` methods to `Meter` interface.
- adds `Int64Gauge` and `Float64Gauge` interfaces
- adds `*GaugeConfig` structs, factory functions, options...